### PR TITLE
Get-Error

### PR DESCRIPTION
**Pull request by theJasonHelmick**
# PR Summary

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+# Get-Error
+
+## SYNOPSIS
+Gets and displays the most recent error messages from the current session.
+
+## SYNTAX
+
+```
+Get-Error [[-InputObject] <psobject>] [-Newest <Int>] [<CommonParameters>]
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: This syntax does not match what I see in Preview.5. Is it changing the preview.6?

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+## DESCRIPTION
+
+The `Get-Error` cmdlet gets a **PSExtendedError** object that represents the current error details
+from the last error that occurred in the session.
+
+You can use 'Get-Error' to display a specified number of errors that have occurred in the current
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: Use backticks = `Get-Error`

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+session using the **Newest** parameter.
+
+The 'Get-Error' cmdlet also receives error objects from a collection, such as **$Error**, to display
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: Use backticks = `Get-Error` and `$Error`. Fix these throughout the article.

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+## EXAMPLES
+
+### Example 1: Get the most recent error details
+
+In this example, `Get-Error` displays the details of the most recent error that occurred in the
+current session.
+
+```powershell
+Get-Childitem -path c:\NoRealDirectory
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: What about making this non-Windows specific? 

```
Get-Childitem -path /NoRealDirectory
```

This works on all three OS targets.

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Pipeline input
+
+`Get-Error` accepts pipeline input. For example, `$Error | Get-Error`.
+
+## OUTPUTS
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: Output is a PSExtendedError object

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+## NOTES
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: We might want to add a note about piping the output to a non-PS command results in the PSExtendedError being converted to string output that contains ANSI escape sequences. We should talk to @SteveL-MSFT about a way to suppress colorized output.

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+## RELATED LINKS
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: We should think about linking to something. We don't have any really good error handling articles except for about_Try_Catch_Finally.

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md (hunk 6 above)
Comment (theJasonHelmick) [reply reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: All of us discussed - I believe we agreed to hold on this till after release and we have received some feedback.  Is that correct Sean?

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md (hunk 7 above)
Comment (theJasonHelmick) [reply reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: Sean, I'm not sure of the reference -- so please check this one again.

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md (hunk 5 above)
Comment (sdwheeler) [reply reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: This is not formatted correctly per the PlatyPS schema. See https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md

File: reference/7/Microsoft.PowerShell.Utility/Get-Error.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+Supports input from any PSObject, but results will vary unless an ErrorRecord or Exception object
+are supplied.
+
+### Pipeline input
```
Comment (sdwheeler) [inline reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: This is not formatted correctly per the PlatyPS schema. See https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md

Comment (doctordns) [reply reference/7/Microsoft.PowerShell.Utility/Get-Error.md]: I'd have liked that sentence to have read:

Supports input from any PSObject, but results vary unless either an ErrorRecord or Exception object
are supplied.

Future tense in documentation is to be avoided.

**Pull request merged**